plugins/utils/spectre.nix

```
@@ -30,6 +30,7 @@ helpers.neovim-plugin.mkNeovimPlugin config {
           (
             with types;
             attrsOf (submodule {
+              freeformType = types.attrs;
```

```suggestion
              freeformType = with types; attrsOf anything;
```

This isn't strictly necessary if we know that "engine" options will always be a **flat** attrset, but the `anything` type allows for **recursive** merging of definitions, while defining nested attrs in multiple locations could clobber each other when using `types.attrs`.

I'll let you decide which type is more appropriate here, `attrs` or `attrsOf anything`.

---

I'd rather go with attrsOf anything, as `attrs` is soft-deprecated in nixpkgs I believe

---

Thanks, updated!